Dockerfile

```
@@ -0,0 +1,13 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env
+
+# Set environment variables for ExampleApp
+ENV DWOLLA_APP_KEY=YOUR_APP_KEY
+ENV DWOLLA_APP_SECRET=YOUR_APP_SECRET
+
+RUN mkdir app && mkdir app/Dwolla.Client && mkdir app/ExampleApp
```

```suggestion
RUN mkdir app app/Dwolla.Client app/ExampleApp
```

Inlining multiple command executions into one.